<launch>

  <arg name="uav_name" default="$(optenv UAV_NAME uav)"/>

  <arg name="debug" default="false" />
  <arg unless="$(arg debug)" name="launch_prefix" value=""/>
  <arg     if="$(arg debug)" name="launch_prefix" value="pdb"/>

  <node name="tf_correction" pkg="uav_localize" type="tf_corrector.py" output="screen" launch-prefix="$(arg launch_prefix)">
    <param name="static_x" value="0" />
    <param name="static_y" value="0" />
    <param name="static_z" value="0" />
    <param name="frame_id" value="fcu_$(arg uav_name)" />
    <remap from="~input_odom" to="/$(arg uav_name)/odometry/odom_main" />
  </node>

  <!-- <group ns="$(arg uav_name)"> -->
  <!--   <node name="img_repub_depth" pkg="image_transport" type="republish" args="compressedDepth in:=/uav81/rs_d435/aligned_depth_to_color/image_raw raw out:=/uav81/rs_d435/aligned_depth_to_color/image_raw" /> -->
  <!--   <node name="img_repub_image" pkg="image_transport" type="republish" args="compressed in:=/uav81/rs_d435/color/image_raw raw out:=/uav81/rs_d435/color/image_rect_color" /> -->
  <!-- </group> -->

</launch>

